src/OrbitGl/Track.cpp

```diff
@@ -81,6 +81,21 @@ void Track::DrawTriangleFan(Batcher& batcher, const std::vector<Vec2>& points, c
   }
 }
 
+void Track::UpdatePositionOfCollapseToggle() {
```

Comment:
Agree, sorry for that. I had to do quite some re-arranging of PRs since this evolved over time and couldn't split it up into smaller pieces afterwards.